frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -441,3 +441,41 @@ action.GitMachete.CheckoutPreviousAction.undefined.branch-name=No branches befor
 action.GitMachete.CheckoutParentAction.text=Git Machete: Checkout Parent Branch
 action.GitMachete.CheckoutParentAction.description=Checkout parent branch (''{0}'')
 action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') does not have a parent in the branch layout
+
+# Traverse
+
+action.GitMachete.TraverseRepositoryAction.description.text=Starting To Traverse
+action.GitMachete.TraverseRepositoryAction.description.empty-layout=Cannot traverse an empty branch layout
+action.GitMachete.TraverseRepositoryAction.GitMacheteToolbar.text=Traverse
+action.GitMachete.TraverseRepositoryAction.invoked-action.id=GitMachete.{0}
+
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.title=Traverse
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.text.HTML=<html>Do you want to traverse the branches of this layout?</html>
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.ok-text=traverse
+
+action.GitMachete.TraverseRepositoryAction.dialog.do-not-show-again=Don't show it again
+
+
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.text.HTML=<html><p>The branch <b>{0}</b> is divereged from remote <b>{1}</b>, and it is <b>{2}</b> remote.</p><p>Please choose the preferred action for syncing it:</p></html>
```

Comment:
dont we need some preposition before the ` remote.`?

Reply:
oh, no, it is `behind remote` or `ahead of remote` I think. So no `to`